Port `before_v0.60/fuzzy`, `before_v0.60/ls-mods` and `before_v0.60/nu_101`

39555 opened the pull request:
This PR is part of porting all old scripts #221 and includes a set of modules:

- fuzzy -> `modules/fuzzy/fuzzy_command_search.nu`
- ls-mods -> `modules/ls-mods`: `ls-less.nu`, `ls-wide.nu` and `ls-wide-with-color.nu`
- nu_101 -> `modules/nu_101`: `nothing.nu` and `inner_outer_loop.nu`

Edit: `fuzzy` and `nu_101` have been moved to `sourced`

fdncred commented:
i noticed some files are just moved over without changes. does that mean they already work or are you continuing to work on these?

39555 commented:
I found that I should move `nu_101` to `sourced`. I will fix that

39555 commented:
Fix missing files done.
also I've merged `nu_101` scripts with `nu_101` in `sourced` and moved `fuzzy` module to `sourced`

## All 11 files that have been changed in this PR:

### `before_v0.60/fuzzy` -> `sourced/fuzzy`

- `README.md` I've moved it from `before_v0.60` and removed mentions about config.toml
- `fuzzy_command_search.nu`
- `fuzzy_history_search.nu` does not need any changes

### `before_v0.60/nu_101` -> `sourced/nu_101`
- `README.md` removed, already in `sourced/nu_101`
- `demo.nu` removed, already in `sourced/nu_101`
- `nothing.nu`
-  `inner_outer_loop.nu`

### `before_v0.60/ls-mods` -> `modules/ls-mods`
- `ls-hidden.nu` was missing the first time. Now I've ported it.
- `ls-less.nu`
- `ls-wide-with-color.nu`
- `ls-wide.nu`

fdncred commented:
modules need to have export, sourced do not

39555 commented:
I moved `fuzzy` into `modules/`, because it contains fuzzy related functions and functions that need to be private and added `export` in `ls-mods` and `fuzzy`

fdncred commented:
thanks
